tests/store/test_sqlalchemy_store.py

```diff
@@ -1106,3 +1107,62 @@ def test_log_batch_same_metric_repeated_multiple_reqs(self):
         self._verify_logged(run.info.run_id, params=[], metrics=[metric0], tags=[])
         self.store.log_batch(run.info.run_id, params=[], metrics=[metric1], tags=[])
         self._verify_logged(run.info.run_id, params=[], metrics=[metric0, metric1], tags=[])
+
+
+class TestSqlAlchemyStoreSqliteMigratedDB(TestSqlAlchemyStoreSqlite):
+    """
+    Test case where user has an existing DB with schema generated by InitialBase,
+    then migrates their DB.
+    """
+    def setUp(self):
+        _, self.tmpfile = tempfile.mkstemp()
+        self.db_url = "sqlite:///%s" % self.tmpfile
+        engine = sqlalchemy.create_engine(self.db_url)
+        InitialBase.metadata.create_all(engine)
+        invoke_cli_runner(mlflow.db.commands, ['upgrade', self.db_url])
+        self.store = SqlAlchemyStore(self.db_url, ARTIFACT_URI)
+
+    def tearDown(self):
+        os.remove(self.tmpfile)
+
+
+@pytest.mark.release
```

Comment:
This test runs & passes locally against MySQL 5.7 (interestingly our DB schema is & has been invalid in MySQL 8.0 because both the runs & experiments table include a constraint named `lifecycle_stage`, but that's a separate problem - will make a GitHub issue to track it)